backend/src/main/java/mouda/backend/chat/controller/ChatSwagger.java

```diff
@@ -0,0 +1,36 @@
+package mouda.backend.chat.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import mouda.backend.chat.dto.request.ChatCreateRequest;
+import mouda.backend.chat.dto.response.ChatFindUnloadedResponse;
+import mouda.backend.common.RestResponse;
+import mouda.backend.config.argumentresolver.LoginMember;
+import mouda.backend.member.domain.Member;
+
+public interface ChatSwagger {
+
+	@Operation(summary = "채팅 생성", description = "한 건의 채팅 메시지를 보낸다.")
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "채팅 생성 성공!")
+	})
+	ResponseEntity<Void> createChat(
+		@RequestBody ChatCreateRequest chatCreateRequest,
+		Member member
```

Comment:
```suggestion
		@LoginMember Member member
```
인터페이스에선 없어도 되네요!🫢 그래도 통일성을 위해서 `@LoginMember`를 추가하는 건 어떤가용?

Reply:
앟 놓쳤었네요 😃😃 감사합니다!!